media/css/firefox/choose.less

```diff
@@ -0,0 +1,179 @@
+@import "../sandstone/lib.less";
+
+#outer-wrapper {
+    background: none;
+}
+#wrapper {
+    width: 100%;
+}
+#masthead {
+    background-color: #0c99d5;
+
+    h2 {
+        padding: 0 0 50px;
+    }
+
+    @media only screen and (min-width: @breakTablet) and (max-width: @breakDesktop) {
+        padding: 0;
+        width: 100%;
+    }
+}
+.header-content {
+    margin: 0 auto;
+    padding: 0 0 80px;
+    width: 90%;
+    max-width: 980px;
+    text-align: center;
+
+    @media only screen and (max-width: @breakMobileLandscape) {
+        padding: 0 0 60px;
+        width: 100%;
+    }
+
+    h1 {
+        margin: 0 10px;
+        padding: 30px 0 20px;
+        text-align: left;
+    }
+
+    h2 {
+        color: #fff;
+        margin: 0 0 55px 0;
+        .open-sans-light();
+        .font-size(4rem);
+        line-height: 1;
+        letter-spacing: -0.035em;
+        text-shadow: none;
+
+        @media only screen and (max-width: 540px) {
+            .font-size(2rem);
+        }
+    }
+    .download-button.download-button-simple {
+        color: #fff;
+        margin-bottom: 0;
+        border: none;
+        font-weight: bold;
+        .font-size(1.5rem);
+        text-align: center;
+        .inline-block();
```

Comment:
I believe this may be superfluous since `.inline-block` is already applied in `buttons.less`